openslides/agenda/static/js/agenda/site.js

```diff
@@ -30,6 +30,9 @@ angular.module('OpenSlidesApp.agenda.site', [
                 url: '/agenda',
                 abstract: true,
                 template: "<ui-view/>",
+                data: {
+                    title: 'Agenda',
```

Comment:
I think this string has to be marked for translation (even if the same string is marked somewhere elese).